### PR TITLE
Reaparing size of class window

### PR DESCRIPTION
**Pull request by KrzucH**

### CODE REVIEW

File: content/MainScreeen.ui.qml
Comment (llisnikowski) [inline content/MainScreeen.ui.qml:1]: Please change `height: Constants.height / 2` to 'height: Constants.height' and change height in Constants.qml file

File: content/MenuButton.qml
Comment (llisnikowski) [inline content/MenuButton.qml:1]: Maybe it's worth using the Column type here.
https://doc.qt.io/qt-6/qml-qtquick-column.html

**Review by llisnikowski — CHANGES_REQUESTED** [review]
Please rebase branch before merge.

File: content/App.qml [hunk 1]
```diff
@@ -14,11 +14,82 @@ Window {
...
+            id: menuButton
+        }
 
-    MenuButton {
-        id: menuButton
-    }
+        Gauges {
+            id: gauges
+
+            width: parent.width * 0.47
```
Comment (dariannax3) [inline content/App.qml:24]: maybe extract width and height, anchors to new file eg. Constrant.qml and define readonly properties there. imo is better  and easer to change values in one place :)
and this 0.47 value will be hidden

**Review by llisnikowski — APPROVED** [review]

File: content/MenuButton.qml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import QtQuick 2.15
 import QtQuick.Controls 2.15
 
-MainScreeen {
+Column {
+    id:column
+    width:100
+    height: parent.height
+    anchors.centerIn: parent
```
Comment (llisnikowski) [inline content/MenuButton.qml:8]: Move to App.qml

File: content/MenuButton.qml [hunk 3]
```diff
@@ -10,27 +14,31 @@ MainScreeen {
         anchors.topMargin: 100
         text: "3DCar"
     }
+
     Button {
         id: acButton
         anchors.top: parent.top
         anchors.horizontalCenter: parent.horizontalCenter
         anchors.topMargin: 175
```
Comment (llisnikowski) [inline content/MenuButton.qml:22]: Use only `anchors.horizontalCenter: parent.horizontalCenter` and `topPadding`, `spacing` properties for column

**Pull request closed without merging**